### ISSUE TITLE
Speech History does not work well with attempt to cancel speech

### ISSUE THREAD

**Issue by fisher729** [jscholes/nvda-speech-history#18]
Hi.

Speech History does not work well when "Attempt to cancel speech for expired focus events" in Advanced settings is set to "yes", which is the default in NVDA 2021.1.

When this setting is enabled, moving over the affected focused elements in the Speech History results in no speech, and therefore these events cannot be copied.

STR:

1. With the setting set to "yes", Open the NVDA menu.
2. Navigate to Preferences, then Tools.
3. Use Speech History to move to the last two previously spoken speech items.

Expected: You should hear Tools sub menu, followed by Preferences sub menu.
Actual: You hear nothing.

This does not seem to affect copying the items to the clipboard, however.

**Comment by fisher729** [comment]
Works well in the latest version.
